docs/presentation/Customization/SortingRule.md

```diff
@@ -15,60 +15,190 @@ instances are first sorted by the highest priority rule and then the lower prior
+| [`priority`](#attribute-priority)               | No        | `number`                                                             | `1000`                         |
+| [`condition`](#attribute-condition)             | No        | [ECExpression](../hierarchies/ECExpressions.md#rule-condition)       | `""`                           |
+| [`class`](#attribute-class)                     | No        | `SingleSchemaClassSpecification`                                     | All classes in current context |
+| [`isPolymorphic`](#attribute-ispolymorphic)     | No        | `boolean`                                                            | `false`                        |
+| *Sorting*                                       |
+| [`propertyName`](#attribute-propertyname)       | Yes       | `string`                                                             |                                |
+| [`sortAscending`](#attribute-sortascending)     | No        | `boolean`                                                            | `true`                         |
+
+### Attribute: `requiredSchemas`
+
+> **Default value:** `[]`
+
+A list of ECSchema requirements that need to be met for the rule to be used. See more details in [Defining ECSchema Requirements for Presentation Rules](../Advanced/SchemaRequirements.md).
+
+```ts
+[[include:Sorting.RequiredSchemas.Ruleset]]
+```
+
+| `minVersion` value   | Result                                                                             |
+| -------------------- | ---------------------------------------------------------------------------------- |
+| `minVersion = 1.0.2` | ![Example of using required schemas condition](./media/sortingrule-sorted-asc.png) |
+| `minVersion = 2.0.2` | ![Example of using required schemas condition](./media/sortingrule-unsorted.png)   |
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order in which rules are handled - higher priority means the rule is handled first. If priorities are equal, the rules are handled in the order they're defined.
+
+```ts
+[[include:PropertySortingRule.Priority.Ruleset]]
+```
+
+![Example of using priority attribute](./media/sortingrule-sorted-asc.png)
+
+### Attribute: `condition`
+
+> **Default value:** `""`
+
+Defines a condition which needs to be met in order for the rule to be used. The condition is an [ECExpression](./ECExpressions.md#rule-condition) which has to evaluate to a boolean value.
+
+```ts
+[[include:PropertySortingRule.Condition.Ruleset]]
+```
+
+| Condition values | Result                                                                 |
+| ---------------- | ---------------------------------------------------------------------- |
+| `TRUE`           | ![Example of using rule condition](./media/sortingrule-sorted-asc.png) |
+| `FALSE`          | ![Example of using rule condition](./media/sortingrule-unsorted.png)   |
+
```

Comment:
This is giving off an impression that `condition` can only be populated with either `TRUE` or `FALSE` values, when in reality this value can be any ECExpression. Maybe label the first column "Condition evaluation result"? Same applies to the other `condition` table too.